sql-to-dbsp-compiler/lib/sqllib/src/string.rs

```diff
@@ -318,3 +320,217 @@ pub fn writelog<T: std::fmt::Display>(format: String, argument: T) -> T {
     print!("{}", formatted);
     argument
 }
+
+pub fn md5__(value: String) -> String{
+    let mut  hasher = Md5::new();
+    hasher.input(value.as_bytes());
+    let result = hasher.result();
+    return format!("{:x}", result);
+}
+
+pub fn to_lower_hex(value: i64) -> String{
+    format!("{:x}", value)
+}
+
+pub fn to_upper_hex(value: i64) -> String{
+    format!("{:X}", value)
+}
+
+pub fn translate_(value: String, from: String, to: String) -> String{
```

Comment:
I don't understand what this function does, a comment would be good

Reply:
it seems to do some form of string replacement, is there a std library function that does the same thing?